src/api/todos.ts

```diff
@@ -1,10 +1,18 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const USER_ID = 0;
+export const USER_ID = 2042;
 
 export const getTodos = () => {
   return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
 };
 
+export const addTodo = (todoData: Todo) => {
+  return client.post<Todo[]>(`/todos`, todoData);
```

Comment:
```suggestion
  return client.post<Todo[]>('/todos', todoData);
```

Reply:
Done) I've changed the next file with the same code